ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -597,6 +597,7 @@ objects:
         }
 
         function setup-google-cloud-sdk() {
+          curwd="$(pwd)"
```

Comment:
suggestion:might be a good time to use `pushd` & `popd`

Reply:
thx, fixed